src/sec_security_openssl.c

```diff
@@ -4930,6 +4937,8 @@ Sec_Result SecKey_ComputeBaseKeyDigest(Sec_ProcessorHandle* secProcHandle, SEC_B
                     &base_key))
     {
         SEC_LOG_ERROR("SecKey_GetInstance failed");
+        SecKey_Release(base_key);
+        base_key = NULL;
```

Comment:
Same as the previous digest initialization.  If SecKey_GetInstance fails, nothing should be allocated, and should not require a release call.